__tests__/lib/messageUtils.test.ts

```diff
@@ -56,3 +56,31 @@ describe('toBinary', () => {
     expect(decodedResult).toEqual(nonLatinString);
   })
 })
+
+describe('getDashboard', () => {
+  // Test that get dashboard returns a correct dashboard links for a given
+  // for a given messageId and template === "infobar".
+  it('returns a correct infobar dashboard link', () => {
+    const template = "infobar"
+    const msgId = "123"
+    const channel = "rel:ease"
```

Comment:
nit: lets add a note that extra characters are intentional to check the encoding